Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -783,13 +788,14 @@ extension NavigationViewController: NavigationServiceDelegate {
             navigationMapView?.moveUserLocation(to: location, animated: true)
         }
 
-        attemptToHighlightBuildings(progress, with: location)
+        attemptToHighlightBuildings(progress)
         
         // Finally, pass the message onto the `NavigationViewControllerDelegate`.
         delegate?.navigationViewController(self, didUpdate: progress, with: location, rawLocation: rawLocation)
     }
     
     private func checkTunnelState(at location: CLLocation, along progress: RouteProgress) {
+        guard usesDarkStyleWhileInTunnel else { return }
```

Comment:
If the check of `usesDarkStyleWhileInTunnel` before the judgment of the `traversingTunnel` and `inTunnel`, there's a case that the `traversingTunnel` wouldn't be updated. Would it affect the following and later tunnel check if the `usesDarkStyleWhileInTunnel` is changed during navigation?

Reply:
Even though `isTraversingTunnel` used only in this method, there is a chance that we might use it in other places of `NavigationViewController` in future. So yes, I agree that we should assign correct value to it each time. Modified the code.